compiler/resolution/implementForallIntents.cpp

```diff
@@ -230,7 +230,7 @@ static void createShadowVars(DefExpr* defChplIter, SymbolMap& uses,
       // See if we want to prune it.
       bool pruneit = false;
 
-      if (tiIntent == INTENT_REF) {
+      if (tiIntent == INTENT_REF || tiIntent == INTENT_REF_MAYBE_CONST) {
```

Comment:
@vasslitvinov - see this change. It sounds like changing forall intents to pass by `ref` will be hard until we can remove `_toLeader` `_toStandalone` because those use the default intent.